deployment/terraform/cloudbuild-tf-plan.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+# run terraform plan for oss-vdb-test
+# TODO: regenerate the api protobufs?
+- name: gcr.io/oss-vdb/terraform
+  args: ['init', '-no-color']
+  dir:  deployment/terraform/environments/oss-vdb-test
+- name: gcr.io/oss-vdb/terraform
+  args: ['plan', '-no-color']
+  dir:  deployment/terraform/environments/oss-vdb-test
+# TODO(michaelkedar): plan for oss-vdb
+
+# Set in Cloud Build:
+# serviceAccount: 'projects/oss-vdb/serviceAccounts/deployment@oss-vdb.iam.gserviceaccount.com'
+options:
+  logging: CLOUD_LOGGING_ONLY
```

Comment:
Personally, I find Cloud Logging output hard to read.
We could make a logging bucket for this to make it [look like this](https://pantheon.corp.google.com/cloud-build/builds;region=global/950ef9cc-1e27-4eff-aa65-558f3fc86545?project=oss-vdb).

I'm not really sure how much you'd have to manage it / delete old logs.

Reply:
That seems a bit better than having to read Cloud Logging logs. Re deleting old logs, we can just set a lifecycle policy: https://cloud.google.com/storage/docs/lifecycle

Reply:
Moved logs to the terraform bucket & made a lifecycle policy to delete after 60 days